examples/ruby/ruby_client.rb

```diff
@@ -0,0 +1,145 @@
+
+CERT_DIR = 'config/https'
+SSL_CERT_FILE = 'ca.crt'
+CONJUR_CERT_FILE = 'conjur.crt'
+CONJUR_KEY_FILE = 'conjur.key'
+
+ACCOUNT="dev"
+LOGIN="admin"
+ADMIN_API_KEY=ENV["CONJUR_ADMIN_API_KEY"]
+
+# Setup client configuration
+config = OpenapiClient.configure
+config.scheme = "https"
+config.host = "localhost"
+# config.debugging = true
+config.username = LOGIN
+config.password = ADMIN_API_KEY
+config.verify_ssl = true
+config.ssl_ca_cert = File.join(CERT_DIR, SSL_CERT_FILE)
+config.cert_file = File.join(CERT_DIR, CONJUR_CERT_FILE)
+config.key_file = File.join(CERT_DIR, CONJUR_KEY_FILE)
+
+# Authenticating admin using basicAuth
+authn_instance = OpenapiClient::AuthnApi.new
+token = nil
+puts "Authenticating admin..."
+begin
+  token = authn_instance.authenticate(account=ACCOUNT, login=LOGIN, body=ADMIN_API_KEY, opts={accept_encoding: 'base64'})
+  puts "Base64 encoded token: #{token}"
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling AuthnApi->authenticate: #{e}"
+  exit(1)
+end
+
+# Change admin password using basicAuth
+new_password = "N3w-Passw0rd!"
+puts "\nChanging admin password..."
+begin
+  authn_instance.set_password(account=ACCOUNT, body=new_password)
+  puts "Password change successful."
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling AuthnApi->set_password: #{e}"
+  exit(1)
+end
+
+config.password = new_password
+
+# Add Conjur Token header to api configuration
+token_body = 'token="%s"' % [token]
+config.api_key['Authorization'] = token_body
+config.api_key_prefix['Authorization'] = 'Token'
+
+policy_instance = OpenapiClient::PoliciesApi.new
+authn_instance = OpenapiClient::AuthnApi.new
+
+# Load empty policy, allows the example to be run multiple times sequentially
+# Loading a policy returns data for users CREATED when the policy is loaded. Without loading 
+# an "empty" policy, if the user alice already exists due to a prior example run, loading 
+# the full policy will not respond with alice's api key.
+puts "\nLoading empty root policy..."
+empty_policy = "- !user admin"
+empty_results = nil
+begin
+  empty_results = policy_instance.load_policy(account=ACCOUNT, identifier="root", body=empty_policy)
+  puts "Empty policy loaded."
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling PoliciesApi->load_policy: #{e}"
+  exit(1)
+end
+
+puts "\nLoading root policy..."
+policy = """---
+- !user admin
+- !user alice
+- !variable sampleSecret
+
+- !permit
+  role: !user admin
+  privilege: [ update ]
+  resource: !user alice
+
+- !permit
+  role: !user admin
+  privilege: [ execute ]
+  resource: !variable sampleSecret
+"""
+loaded_results = nil
+begin
+  loaded_results = policy_instance.load_policy(account=ACCOUNT, identifier="root", body=policy)
+  puts "Policy loaded."
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling PoliciesApi->load_policy: #{e}"
+  exit(1)
+end
+
+alice_api_key = loaded_results[:created_roles][:"dev:user:alice"][:api_key]
```

Comment:
I think you can use `["dev:user:alice".to_sym]` instead of `[:"dev:user:alice"]`